VogtObserver2/Program.cs

```diff
@@ -1,12 +1,58 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Threading;
 
 namespace VogtObserver2
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            
+
+            Run();
+        }
+
+        public static void Run()
+        {
+            StockGrabber stockGrabber = new StockGrabber();
+
+            Trader t1 = new Trader("Julie");
+            Trader t2 = new Trader("Amy");
+            Trader t3 = new Trader("Mark");
+
+            stockGrabber.AddObserver(t1);
+            stockGrabber.AddObserver(t2);
+            stockGrabber.AddObserver(t3);
+
+
+            for (int i = 0; i <= 5; i++)
+            {
+
+                stockGrabber.FetchStockPrice(GetRandomStockSymbol());
+
```

Comment:
Awesome this looks a lot better! The progrma creates a few traders, has them start observing the stock fetcher, then has the fetcher update itself every 2 seconds 5x. I don't see where you define which stocks they're observing (or if theyre just observing all stock changes). Will keep reviewing